lib/storage/storage.go

```diff
@@ -573,6 +575,54 @@ func (s *Storage) UpdateMetrics(m *Metrics) {
 	s.tb.UpdateMetrics(&m.TableMetrics)
 }
 
+var (
+	isStorageLimitReached      uint32
```

Comment:
It would be better moving this var to `Storage` struct. This will allow opening multiple storage instances in the same app.